### DISCUSSION TITLE
Welcome to picoclock Discussions!

### DISCUSSION THREAD

**Discussion by dawigit** [dawigit/picoclock#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by qw3r3wq** [comment]
Hi,

I am Linux admin, from Lithuania.
Bought rp2040 dev board, trying to learn C or C++ coding, starting from modifying something what is working.

See you!
